extension/src/background.js

```diff
@@ -255,11 +261,11 @@ async function updateState (tab: chrome$Tab) {
           .then(() => chromeTabsInsertCSS(tabId, {code: opts.position_css}));
 
 
-    // NOTE: if the page is unreachable, we can't inject stuf in it
+    // NOTE: if the page is unreachable, we can't inject stuff in it
     // not sure how to detect it? tab doesn't have any interesting attributes
     // firefox sets tab.title to "Server Not Found"? (TODO also see isOk logic below)
     // TODO in this case, could set browser action to open a new tab (i.e. search) or something?
-    await defensify(inject, 'sidebar injection')();
+    await defensify(inject, 'sidebar injection for tabId:' + tabId + ' url: ' + url)();
```

Comment:
nit: generally I prefer writing this as 

    `sidebar injection for tabId: ${tabid} url: ${url}`

, because in many languages if the string is null, `+` would fail, whereas string interpolation would work. Since it's JS either way works though.